packages/core/src/LottieView/index.web.tsx

```diff
@@ -24,151 +20,128 @@ const LottieView = forwardRef(
       hover,
       direction,
       progress,
+      onAnimationLoaded,
       onAnimationFailure,
       onAnimationFinish,
       onAnimationLoop,
     }: LottieViewProps,
-    ref: any
+    ref: any,
   ) => {
+    const [dotLottie, setDotLottie] = useState<DotLottie>(null);
```

Comment:
Shouldn't this be DotLottie | null?

Reply:
Yes, typescript won't complain in this project because `strict` is false in the tsconfig.

I made the necessary changes for the web implementation to be compliant with strict mode should we choose to enable it.